src/tests/Microsoft.PowerFx.Interpreter.Tests/MutationScripts/Patch.txt

```diff
@@ -33,23 +33,20 @@ Table({b:true,d:1,t:"hi"},{b:false,d:2,t:"back"},{b:Blank(),d:3,t:"cycle"})
 Table({b:true,d:1,t:"hi"},{b:Blank(),d:12,t:"different"},{b:Blank(),d:3,t:"cycle"})
 
 >> Patch( y, Index(y,2), {b:1,d:"23",t:DateTime(2002,2,2,2,2,2)})
-{b:true,d:23,t:"2/2/2002 2:02 AM"}
+Errors: Error 22-61: The type of this argument 'd' does not match the expected type 'Decimal'. Found type 'Text'.|Error 0-62: The function 'Patch' has some invalid arguments.
```

Comment:
This would be a good time for us to start logging at least semantic breaking changes...

Reply:
Let's bring this subject to our next team meeting.

Reply:
Yes.  We should come up with a way to track.
The change from runtime to compile time error is intended.